docs/variants/msi_z690/cpu-hcl.md

```diff
@@ -2,41 +2,158 @@
+  report automatically.
+* Create new issue in [Dasharo issues
+  repository](https://github.com/Dasharo/dasharo-issues/issues).
+* Create PR directly to [Dasharo documentation
+  repository](https://github.com/Dasharo/docs).
+* [email](mailto:contact@dasharo.com?subject=CPU HCL report) you
+  `/proc/cpuinfo` or relevant information using following template:
+
+  ```text
+  CPU model:
+  Dasharo version:
+  ```
 
 If you already have reported your results and you change some hardware
-configuration we would appreciate an additional HCL report.
-
-## HCL list
-
-CPU Hardware Compatibility List presents the CPUs tested and verified
-to work with Dasharo by community. The following list does not include CPU
-which is tested and verfied in 3mdeb laboratory - this information might be
-found in [Hardware Matrix](hardware-matrix.md) documentation.
-
-Legend:
-
-* Processor name: the full name of processor including vendor, brand and
-    CPU number.
-* Core name: CPU core codename.
-* CPU base speed: based CPU speed.
-* CPU boost speed: boosted CPU speed.
-* Wattage: declared by producer processor wattage.
-* GPU: information about embedded graphics processing unit.
-* Results: link to measurement results.
-
-Information about processor name, core name and speed might be read from OS
-(Linux systems - command `lscpu`, Windows systems - `System information` menu).
-Rest of the information might be read from CPU package or documentation.
-
-| Processor name       | Core name   | CPU base speed | CPU boost speed | Wattage | GPU             | Results                |
-|:--------------------:|:-----------:|:--------------:|:---------------:|:-------:|:---------------:|:----------------------:|
-| Intel Core i9-12900K | Alder Lake  | 3.2 GHz        | 5.3 GHz         | 125     | UHDGraphics710  | [Qubes HCL reports][1] |
-| Intel Core i7-12700K | Alder Lake  | 3.6 GHz        | 5.0 GHz         | 125     | UHDGraphics770  | [Qubes HCL reports][2] |
-| Intel Core i5-12400F | Alder Lake  | 2.5 GHz        | 4.4 GHz         | 65      | -               |                        |
-
-[1]: <https://forum.qubes-os.org/t/msi-pro-z690-a-wifi-ddr4-with-alder-lake-12900k/11490/6>
-[2]: <https://www.qubes-os.org/hcl/#msi_ms-7d25_i7-12700k_alder-lake_integrated-graphics-uhd-770-geforce-gtx-1080-ti>
+configuration we would appreciate an update.
+
+## HCL lists
+
+=== "CPU HCL"
+
+    CPU Hardware Compatibility List presents the CPUs tested and verified
+    to work with Dasharo by community. The following list does not include CPU
+    which is tested and verfied in 3mdeb laboratory - this information might be
+    found in [Hardware Matrix](hardware-matrix.md) documentation.
+
+    Legend:
+
+    * CPU Model: Intel CPU Model name.
+    * Dasharo version: Dasharo version on which report was created.
+        - `-` if version was not reported
+    * Source:
+        - Link to report if it is public.
+        - `Dasharo HCL report` if it was reported using DTS.
```

Comment:
We may link to https://docs.dasharo.com/dasharo-tools-suite/documentation/#hcl-report so someone new can find more information about that

Reply:
We already link to it in Contributing section, but I fixed it anyway: https://github.com/Dasharo/docs/pull/463/commits/514e3ce2d40fc8f007b7eb62b312826352d7a693